src/config.js

```diff
@@ -0,0 +1,10 @@
+import AWS from "aws-sdk";
+
+AWS.config.update({
+  region: "ap-south-1",
+  credentials: {
+    accessKeyId: "AKIAJO47LOOLEI26LN3A",
+    secretAccessKey: "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"
```

Comment:
there should be one global config env file. if this is that then fine. 

but something like

.env/DEV
.env/PROD 
is expected with webpack. discuss if not clear with me